packages/embark-console/src/lib/index.ts

```diff
@@ -124,21 +124,21 @@ class Console {
         __("possible commands are:"),
         // TODO: only if the blockchain is actually active!
         // will need to pass te current embark state here
-        "ipfs".cyan + " - " + __("instantiated js-ipfs object configured to the current environment (available if ipfs is enabled)"),
-        "swarm".cyan + " - " + __("instantiated swarm-api object configured to the current environment (available if swarm is enabled)"),
-        "web3".cyan + " - " + __("instantiated web3.js object configured to the current environment"),
-        "EmbarkJS".cyan + " - " + __("EmbarkJS static functions for Storage, Messages, Names, etc."),
-        "log [process] on/off".cyan + " - " + __("Activate or deactivate the logs of a sub-process. Options: blockchain, ipfs, webserver"),
+        chalk.cyan("ipfs") + " - " + __("instantiated js-ipfs object configured to the current environment (available if ipfs is enabled)"),
+        chalk.cyan("swarm") + " - " + __("instantiated swarm-api object configured to the current environment (available if swarm is enabled)"),
+        chalk.cyan("web3") + " - " + __("instantiated web3.js object configured to the current environment"),
+        chalk.cyan("EmbarkJS") + " - " + __("EmbarkJS static functions for Storage, Messages, Names, etc."),
+        chalk.cyan("log [process] on/off") + " - " + __("Activate or deactivate the logs of a sub-process. Options: blockchain, ipfs, webserver"),
```

Comment:
I think you can just `import('color')` and the original syntax works

Reply:
I assume that one extends `String`?

Reply:
Yes it does. It's an import with side effects